boxsdk/object/base_api_json_object.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from abc import ABCMeta
+
+import six
+
+from boxsdk.util.translator import Translator
+
+
+class BaseAPIJSONObjectMeta(ABCMeta):
```

Comment:
`ABCMeta` is for abstract baseclasses.

For metaclasses, just use `type`. I don't know why it was being used for `ObjectMeta` (probably an accident), but it wasn't necessary.